backend/app/controllers/workflows.js

```diff
@@ -228,10 +228,38 @@ module.exports = {
                     });
                 }
             }
+            // var result = yield * setCurrentStepToNull(req, productId); - not required, as User could require to adjust certain Step's permissions for running Project
+
+            return {
+                updated: updatedIds,
+                inserted: insertIds
+            };
+
+        }).then(function (data) {
+            res.json(data);
+        }, function (err) {
+            next(err);
+        });
+    },
+
+    stepsDelete: function (req, res, next) { //James - I don't know if this works yet. Only stripped it out.
```

Comment:
Can we test this to ensure it works as expected?

Reply:
Not at this time. I've got too much on my plate.